src/backoffice/settings/languages/workspace/language-root/language-root-table-name-column-layout.element.ts

```diff
@@ -0,0 +1,23 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css';
+import { css, html, LitElement, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('umb-language-root-table-name-column-layout')
+export class UmbLanguageRootTableNameColumnLayoutElement extends LitElement {
+	static styles = [UUITextStyles, css``];
+
+	@property({ attribute: false })
+	value!: any;
```

Comment:
If you don't have the type available at the moment, you could always do 
```
@property({ attribute: false })
value: { key: string, name: string };
```

```suggestion
  value!: { key: string, name: string };
```